FAQ/FAQ-General.md

```diff
@@ -130,15 +130,16 @@ If you do not have a store, but want to create one, read [How to Create an Onlin
 
 BTCPay natively supports:
 
-- Bitcoin (BTC)
```

Comment:
no more BTC in btcpay? @pavlenex

Reply:
oops, fixed, how about this toxic @britttttk?